README.md

```diff
@@ -8,6 +8,41 @@ Yet another note taking tool.
 cargo tauri dev
 ```
 
+## Setup environment
```

Comment:
I prefer a rather short "Build and Run" section. See [note.js readme](https://github.com/falk-werner/note.js/tree/feature/update-readme) for reference. We should merge the setup with the previous section, where the build instructions should preceed the run instructions.

Reply:
With commit c337707716c614b8271430b3999e47c3e33cc656 I tried to merge it together without overloading the screen with too much information.
Especially, I hide the steps for installing the dependencies behind a collapsible section to encourage a reader to look up both tauri and vite.